Add search to satellite list

astra137 opened the pull request:
Add a navigation controller and use it to show a search bar above the tableview.

I just really wanted this feature for testing. I have no idea if navigation controllers are the right way to get a search bar like this, but the tutorials online seemed to suggest so.

![image](https://user-images.githubusercontent.com/8324981/76915022-2b98ff00-6879-11ea-84a5-beb920a63c61.png)

Commit:
Add search to satellite list

Add a navigation controller and use it to show a search bar above the tableview.

starburn commented:
It would be nice to search on radio records as well as the name and ID in the TLE.

Commit:
Fix disappearing tracking status

I think the filtered list references can become detached from the Cache.list instances somehow. This is a temporary fix (maybe).

starburn commented:
This is sloppy, but this PR is satisfied by #13
